CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+# How to Contribute to Jaeger
+
+We'd love your help!
+
+Jaeger is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+# Certificate of Origin
```

Comment:
Does this header need to be here? If you want to have a discussion of DCO before the details nearer the end, then possibly don't have the header - or else make it level 2?